src/rgw/store/sfs/CHANGELOG.md

```diff
@@ -15,10 +15,21 @@ and this project adheres to
 
 - Ability to list buckets via admin REST API
 
+- Multipart uploads
+
 ### Fixed
 
 - Show delete markers when listing object versions.
 
+### Notes
+
+- Multipart uploads are currently tracked solely in memory. Should the gateway
+  be stopped, on-going multipart uploads will be lost.
+
+- Aborting on-going multipart uploads is currently not supported.
+
+- Listing multipart uploads does not account for prefix or delimiters.
```

Comment:
I would prefer we keep the existing convention where we are not inserting blank lines between list elements.
Not a real preference, just to respect what is the current convention.